src/prefect/server/api/workers.py

```diff
@@ -372,20 +372,19 @@ async def get_scheduled_flow_runs(
             limit=limit,
         )
 
-    polled_work_queue_ids = [wq.id for wq in work_queues]
-    ready_work_queue_ids = [
-        wq.id for wq in work_queues if wq.status == WorkQueueStatus.NOT_READY
-    ]
-
     background_tasks.add_task(
         mark_work_queues_ready,
-        polled_work_queue_ids=polled_work_queue_ids,
-        ready_work_queue_ids=ready_work_queue_ids,
+        polled_work_queue_ids=[
+            wq.id for wq in work_queues if wq.status != WorkQueueStatus.NOT_READY
+        ],
```

Comment:
this change is not strictly necessary but w/o it the DB has to run an update on last_polled for rows that match these IDs w/ status = NOT_READY _twice_. 
By splitting this list out, it reduces some redundancy.

also matches parity w/ Cloud.